content/applications/productivity/spreadsheet/visualize_data/conditional_formatting.rst

```diff
@@ -0,0 +1,142 @@
+======================
+Conditional formatting
+======================
+
+Conditional formatting applies specific formatting, such as a background color or font color, to
+a range of cells that meet one or more defined criteria, or rules. This allows you to visually
+highlight important data, patterns, trends, or anomalies.
+
+.. note::
+   When a spreadsheet containing dynamic Odoo data is reopened or refreshed, any conditional
+   formatting that has been defined is updated to reflect any changes to the data.
+
+Different types of conditional formatting are possible:
+
+- :ref:`Single color <spreadsheet/conditional-formatting/single-color>`: changes the color of the
+  cell’s background or font, or changes the font formatting if cell values meet the defined
+  conditions, e.g., changes a cell to red if an invoice is overdue.
+- :ref:`Color scale <spreadsheet/conditional-formatting/color-scale>`: uses color intensity to
+  represent smaller and larger values on a two- or three-color scale, e.g., shows sales performance
+  using increasingly dark tones of the same color.
+- :ref:`Icon set <spreadsheet/conditional-formatting/icon-set>`: uses sets of green, orange and red
+  arrows, smileys or dots to visually show how cell values in a range compare to certain defined
+  values, e.g., a green smiley to represent sales performance above a certain amount.
+- :ref:`Data bar <spreadsheet/conditional-formatting/data-bar>`: inserts colored bars inside a cell
+  to show how a value compares to the value of other cells in the defined range, e.g., the customer
+  generating the highest sales has the longest bar, with bars of decreasing length for customers
+  generating lower sales.
+
+To add conditional formatting to a range of cells:
+
+#. Click :menuselection:`Format --> Conditional formatting` from the menu bar.
+#. In the :guilabel:`Conditional formatting` panel that opens to the right of the spreadsheet, click
+   :guilabel:`+ Add another rule`.
+#. Click below :guilabel:`Apply to range`, then, in the spreadsheet, select the range of cells the
+   formatting rule should apply on. Click :guilabel:`Confirm`.
```

Comment:
I suppose most people would begin by selecting the range in the spreadsheet and then applying the formatting, so I would start with that?